subprojects/packagefiles/qarchive/meson.build

```diff
@@ -0,0 +1,72 @@
+project(
+  'QArchive',
+  'cpp',
+  version: '2.1.1',
+  license: 'BSD-3-Clause',
+  default_options: ['cpp_std=c++11'],
+  meson_version: '>=0.49.0')
```

Comment:
It’s recommended to use , and ) on a new line. That’s how muon’s meson formatter does it. Here and elsewhere.